src/migration/9_fix_userdata_columns.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE [dbo].[USERDATA] DROP CONSTRAINT [DF_USERDATA_strSkill]
+
+-- Implicit conversion from data type varchar to binary is not allowed.
+ALTER TABLE [dbo].[USERDATA] ADD [bSkill] BINARY (10) NULL;
+GO
+UPDATE [dbo].[USERDATA] SET [bSkill] = CONVERT(BINARY, [strSkill]);
+GO
+ALTER TABLE [dbo].[USERDATA] DROP COLUMN [strSkill];
+GO
```

Comment:
Note that this will change the columns order in USERDATA table and put `bSkill` as the last column in the table. I know this is a cosmetic thing and we can always specify the order of the columns when we query it for data, but just for the sake of consistency, let me rework this and just recreate the table following the desired order.

Reply:
Done here: https://github.com/ko4life-net/ko-db/pull/12/commits/d5563e3b4cd371986eaa836dff6022ac03d016e7